product-service/src/handlers/catalogBatchProcess.ts

```diff
@@ -0,0 +1,58 @@
+import { SNS } from 'aws-sdk';
+import { PublishInput } from 'aws-sdk/clients/sns';
+import productProviderDynamoDB from '../dynamodb-providers/product.provider';
+import { HttpResponse } from '../helpers/http-response';
+import { isJson } from '../utils/isJson';
+
+const catalogBatchProcess = (event: any, context: any, cb: any) => {
+    const sns = new SNS();
+    const products = event.Records;
+    try {
+        products.forEach(async (data: any) => {
+            const product = isJson(data.messageAttributes)
+                ? JSON.parse(data.messageAttributes)
+                : data.messageAttributes;
+            
+            console.log('Data: ' + JSON.stringify(data));
+            console.log('Product: ' + JSON.stringify(data.body));
+
+            const productNew = {
+                title: product.title.stringValue,
+                description: product.description.stringValue,
+                price: product.price.stringValue,
+                count: product.count.stringValue
+            };
+            const { title, description, price, count } = productNew;
+
+            console.log(title, description, price, count);
+
+            productProviderDynamoDB.createProduct(
+                title,
+                description,
+                price,
+                count
+            );
+
+            const publishParams: PublishInput = {
+                Subject: 'Products has been created',
+                Message: "Success",
+                MessageAttributes: data.messageAttributes,
+                TopicArn: 'arn:aws:sns:eu-west-1:673313573473:createProductTopic',
```

Comment:
[minor]
It's better not to hardcode this things in code, but you can just store it in serverless.ts and pass as environment variables to lambda function (arn would be different for different environments for example)